src/application/scene/fxml/Drawboard.fxml

```diff
@@ -1,10 +1,123 @@
+                           </graphic>
+                           <HBox.margin>
+                              <Insets right="10.0" />
+                           </HBox.margin>
+                        </ToggleButton>
+                        <Button fx:id="buttonSave" mnemonicParsing="false" onAction="#save" prefHeight="60.0" prefWidth="60.0" styleClass="button_tool">
+                           <graphic>
+                              <ImageView fitHeight="30.0" fitWidth="30.0" pickOnBounds="true" preserveRatio="true">
+                                 <image>
+                                    <Image url="@../assets/icon_drawboard_diskette.png" />
+                                 </image>
+                              </ImageView>
+                           </graphic>
+                           <HBox.margin>
+                              <Insets right="10.0" />
+                           </HBox.margin>
+                        </Button>
+                        <ColorPicker fx:id="colorPicker" cache="true" prefHeight="60.0" prefWidth="60.0" stylesheets="@../css/Drawboard.css">
+                           <value>
+                              <Color red="1.0" />
+                           </value>
+                           <styleClass>
+                              <String fx:value="button_tool" />
+                              <String fx:value="color_picker" />
+                           </styleClass>
+                           <HBox.margin>
+                              <Insets right="5.0" />
+                           </HBox.margin>
+                        </ColorPicker>
+                     </children>
+                  </HBox>
+               </children>
+            </AnchorPane>
+         </top>
+         <center>
+            <Pane maxHeight="750.0" maxWidth="1315.0" prefHeight="750.0" prefWidth="1315.0" style="-fx-background-color: white;" BorderPane.alignment="TOP_CENTER">
+               <children>
+                  <Canvas fx:id="canvas" height="750.0" width="1315.0" />
+               </children>
+            </Pane>
+         </center>
+      </BorderPane>
+      <AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="263.0" prefWidth="54.0" style="-fx-background-color: #332E43; -fx-background-radius: 3px;" stylesheets="@../css/Drawboard.css" AnchorPane.bottomAnchor="250.0" AnchorPane.rightAnchor="0.0">
+         <children>
+            <Slider fx:id="sliderToolSize" layoutX="6.0" min="1.0" orientation="VERTICAL" prefHeight="220.0" styleClass="slider_tool" stylesheets="@../css/Drawboard.css" value="5.0" AnchorPane.bottomAnchor="43.0" AnchorPane.topAnchor="0.0" />
+            <ImageView fx:id="toolIndicater" fitHeight="35.0" fitWidth="35.0" layoutX="10.0" layoutY="220.0" pickOnBounds="true" preserveRatio="true" style="-fx-opacity: 0.6;">
+               <image>
+                  <Image url="@../assets/icon_tool.png" />
```

Comment:
I found this settings button a little misleading, because I thought that clicking it would lead to more options (rather than just being an indicator of the panel usage. Is there something that it could be changed to that would be more intuitive (I know on photoshop it has a little circle icon)

Reply:
setting button has been removed and updated with a visualizer of brush size in pixels

Reply:
Noted, the gear icon was removed below the tool size slider and instead replaced with a dynamically updating visualizer of brush size